### PR TITLE
Some proposals for the Infidel role

### PR DESCRIPTION
**Pull request by Tomsod**
Hello again!  I finally got around to updating my version of the Infidel Patch with all the bugfixes and some cool ideas of your implementation.  In the process, I had a few ideas of my own and hereby present them to you.  Pick any commits you like, or edit them, or discuss, these are but proposals.

### CODE REVIEW

**Comment by k21971** [comment]
Hiya Tomsod! These all look very interesting. I'll review these more in-depth later today. Thanks for the PR.

File: src/pray.c [hunk 1]
```diff
@@ -1006,7 +1006,14 @@ gcrownu()
         break;
     }
     case A_NONE:
-        /* nothing to do */
+        /* OK, we don't get an artifact, but surely Moloch
+         * can at least offer His own blessing? */
+        if (ok_wep(obj) && !obj->oartifact && !(obj->oprops & ITEM_FIRE))
```
Comment (Tomsod) [inline src/pray.c:1011]: Oops, just realized the player can wield stackable weapons.  50 daggers of fire is probably too much!

Comment (Tomsod) [reply src/pray.c]: And fixed.  Hopefully my force-push hadn't exploded anyone's PC.

**Review by k21971 — APPROVED** [review]

**Review by k21971 — APPROVED** [review]

**Review by k21971 — APPROVED** [review]

**Review by k21971 — APPROVED** [review]
This is awesome. I am horrible at making tiles, glad you did this :)

**Review by k21971 — APPROVED** [review]

**Review by k21971 — APPROVED** [review]

**Pull request merged**